src/ray/gcs/gcs_server/gcs_monitor_server.h

```diff
@@ -14,13 +14,18 @@
 
 #pragma once
 
+#include "ray/gcs/gcs_server/gcs_kv_manager.h"
 #include "ray/gcs/gcs_server/gcs_node_manager.h"
 #include "ray/gcs/gcs_server/gcs_placement_group_manager.h"
 #include "ray/rpc/gcs_server/gcs_rpc_server.h"
 
 namespace ray {
 namespace gcs {
 
+// NOTE: This needs to be kept in sync with ray.autoscaler._private.constants.
+const std::string AUTOSCALER_SDK_REQUEST_RESOURCES_KEY =
```

Comment:
Ideally, we should use cython to enforce this.

And this seems wrong to me. If this file is included twice, there should be error like duplicate definition?

constexpr char

or just a global variable with declaration in header and implementation in cpp.

Reply:
Not relevant once the new endpoint is introduced (becomes an implementation detail of the class)